### PR TITLE
Update exit.on.started value in 1.4

### PR DESCRIPTION
**Pull request by batsatt**
Replace use of unicode character with `'!'` since it doesn't work on some systems.

### CODE REVIEW

File: applications/mp/pom.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
     <parent>
         <groupId>io.helidon.applications</groupId>
         <artifactId>helidon-applications-project</artifactId>
-        <version>1.4.1-SNAPSHOT</version>
+        <version>1.4.1</version>
```
Comment (barchetta) [inline applications/mp/pom.xml]: Danger! Danger!

Comment (batsatt) [reply applications/mp/pom.xml]: Huh, not sure what happened here. Will merge.

Comment (batsatt) [reply applications/mp/pom.xml]: Done.

Comment (batsatt) [reply applications/mp/pom.xml]: I don't like having that merge here, going to redo since it is such a simple change.

**Pull request closed without merging**